app/pages/dashboard/PatientDrawerContent/useAGPImages/getOpts.js

```diff
@@ -0,0 +1,106 @@
+import moment from 'moment-timezone';
+import _ from 'lodash';
+import get from 'lodash/get';
+import { utils as vizUtils } from '@tidepool/viz';
+
+const getTimezoneFromTimePrefs = vizUtils.datetime.getTimezoneFromTimePrefs;
+
+const AGP_PERIOD_IN_DAYS = 7; // Temporarily using fixed value for prototype launch
+
+const getOpts = (
```

Comment:
To create this function `getOpts`, I started at [this call to onClickPrint()](https://github.com/tidepool-org/blip/blob/WEB-3336-tide-drawer/app/components/PrintDateRangeModal.js#L247) and worked backwards up the callback chain to figure out how the `printOpts` arg is determined.

Would be awesome if you could help me double check this work when we get to code review, I'm a little nervous about it.